sdk/loadtestservice/Azure.Developer.LoadTesting/src/CustomCode/LoadTestingClient.cs

```diff
@@ -0,0 +1,89 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
by typical convention, non-generated client types would go in the root of `/src`

Reply:
changed